AUTHORS

```diff
@@ -34,3 +34,4 @@ Yang Bai <hamo.by@gmail.com>
 Yifa Zhang <zyfdegg@gmail.com>
 Zhanpeng Chen <lowstz@gmail.com>
 Brian Knox <taotetek@gmail.com>
+Maarten Dirkse <mdirkse@bol.com>
```

Comment:
Nit-pick: Should ideally be sorted alphabetically. (Your predecessor is out-of-order too.)